securedrop/journalist_app/utils.py

```diff
@@ -313,18 +309,6 @@ def col_delete(cols_selected: List[str]) -> werkzeug.Response:
     return redirect(url_for('main.index'))
 
 
-def make_password(config: SDConfig) -> str:
-    while True:
-        password = current_app.crypto_util.genrandomid(
-            7,
-            i18n.get_language(config))
-        try:
-            Journalist.check_password_acceptable(password)
```

Comment:
There are 3 checks in `check_password_acceptable()`:

1. Minimum length
2. Maximum length
3. Minimum number of words

They logic implemented in `PassphrasesGenerator.generate_passphrase()` ensures that all generated passphrases are "good" (based on these 3 criteria); it can't return a "bad"/low-quality passphrase. 
The tests on `generate_passphrase()` also double check that.

Reply:
Do you mean that because word lists that could produce shorter or longer passphrases are dropped in `__init__`, `generate_passphrase` doesn't need to check?

Reply:
Exactly - there is no code path where a "bad" passphrase gets generated.